src/modm/platform/dma/stm32/dma.hpp.in

```diff
@@ -375,6 +389,32 @@ struct DmaController<{{ channels.instance }}>::Channel<DmaBase::Channel::Channel
 	%% endfor
 %% endfor
 
+%%else
+%% for channels in dma["channels"]
+	%% for channel in channels.channel
+		%% for signal in channel.signal
+			%% set peripheral = signal.driver.capitalize()
+			%% if signal.instance is defined
+				%% set peripheral = peripheral ~ signal.instance
+			%% else
+				%% if peripheral not in ["Quadspi", "Aes", "Dcmi", "Hash", "Dac"]
+					%% set peripheral = peripheral ~ 1
+				%% endif
```

Comment:
Is this a bug in the modm-devices data?

Reply:
No. Was a (existing) bug in the DMA driver. The else branch should not be there at all and is now removed.